<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1605</width>
    <height>1037</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="RenderWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>1521</width>
      <height>901</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="loadImageButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotateLeftButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate Left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotateRightButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate Right</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zoomInButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoom In</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zoomOutButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoom Out</string>
    </property>
   </widget>
   <widget class="QPushButton" name="panLeftButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>20</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Pan Left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="panRightButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>20</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Pan Right</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1605</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>RenderWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>RenderWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
